x-pack/plugins/security_solution/cypress/fixtures/cidr_list.txt

```diff
@@ -0,0 +1,2 @@
+192.168.100.14/24
+
```

Comment:
Is this extra space here needed so you can split by `\n` here https://github.com/elastic/kibana/pull/83026/files#diff-c2160d09d0ff315067237ed293dda139f3846cf30b7673f77ce239f7e09b3c54R216?

Reply:
No, that's not needed. Thanks, removed it.